src/librbd/cache/rwl/Types.h

```diff
@@ -178,13 +198,14 @@ struct WriteLogPmemEntry {
     : image_offset_bytes(image_offset_bytes), write_bytes(write_bytes),
       entry_valid(0), sync_point(0), sequenced(0), has_data(0), discard(0), writesame(0) {
   }
+  BlockExtent block_extent();
   bool is_sync_point();
   bool is_discard();
   bool is_writesame();
   bool is_write();
```

Comment:
Nit: are these used? Can they be removed and instead just rely on polymorphic methods to handle any deltas in behaviour.

Reply:
I removed the interfaces. Currently, they are not used.